fedocal/fedocallib/__init__.py

```diff
@@ -713,8 +717,19 @@ def add_meeting_to_vcal(ical, meeting):
                 until=meeting.recursion_ends))
         entry.rruleset = newrule
 
+    if enable_reminder:
+        for trigger in (
+                timedelta(**reminder) for reminder in reminders):
```

Comment:
I'd rather have only 1 reminder for a meeting, I don't want to my phone/calendar app to keep buzzing me until I actually have to attend the meeting.

My default would be: remind me 10 or 15 minutes before the meeting